src/main/java/ch/uzh/ifi/seal/soprafs20/board/Board.java

```diff
@@ -140,21 +208,48 @@ public Board move(Figure figure, Field targetField) {
         if (currentField instanceof FirstField && !(targetField instanceof FirstField)) {
             ((FirstField) currentField).setBlocked(false);
         }
+
+        boardRepository.saveAndFlush(this);
```

Comment:
Again board Repository (I think functionality that requires the repo should probably go into a service)